src/Parser/OrientationParser.js

```diff
@@ -0,0 +1,124 @@
+        east.set(-up.y, up.x, 0);
+        east.normalize();
+        north.crossVectors(up, east);
+        elements[0] = east.x; elements[4] = north.x; elements[8] = up.x;
+        elements[1] = east.y; elements[5] = north.y; elements[9] = up.y;
+        elements[2] = east.z; elements[6] = north.z; elements[10] = up.z;
+        return target.setFromRotationMatrix(matrix);
+    };
+})();
+
+/**
+ *
+ * @typedef Attitude
+ * @type {Object}
+ *
+ * @property {Number} omega - angle in degrees
+ * @property {Number} phi - angle in degrees
+ * @property {Number} kappa - angle in degrees
+ * @property {Number} roll - angle in degrees
+ * @property {Number} pitch - angle in degrees
+ * @property {Number} heading - angle in degrees
+ */
+
+
+const ENUQuat = new THREE.Quaternion();
+
+export default {
+
+    /**
+     * @function setFromAttitude
+     * @param {Attitude} attitude - [Attitude]{@link module:OrientedImageParser~Attitude}
+     * with properties: (omega, phi, kappa), (roll, pitch, heading) or none.
+     * Note that convergence of the meridians is not taken into account.
+     * @param {THREE.Quaternion} target Quaternion to set
+     * @returns {THREE.Quaternion} target
+     */
+    setFromAttitude(attitude, target = new THREE.Quaternion()) {
+        if ((attitude.roll !== undefined) || (attitude.pitch !== undefined) || (attitude.heading !== undefined)) {
+            return setFromRollPitchHeading(attitude.roll, attitude.pitch, attitude.heading, target);
+        }
+        if ((attitude.omega !== undefined) || (attitude.phi !== undefined) || (attitude.kappa !== undefined)) {
+            return setFromOmegaPhiKappa(attitude.omega, attitude.phi, attitude.kappa, target);
+        }
+        return target.set(0, 0, 0, 1);
+    },
+
+    /**
+     * @function setFromAttitude
+     * @param {Attitude} attitude - [Attitude]{@link module:OrientedImageParser~Attitude}
+     * with properties: (omega, phi, kappa), (roll, pitch, heading) or none.
+     * @param {Coordinates} coordinate position on the globe
+     * @param {THREE.Quaternion} target Quaternion to set
+     * @returns {THREE.Quaternion} target
+     */
+    setFromAttitudeGlobe(attitude, coordinate, target = new THREE.Quaternion()) {
+        setENUFromGeodesicNormal(ENUQuat, coordinate.geodesicNormal);
+        this.setFromAttitude(attitude, target);
+        target.premultiply(ENUQuat);
+        return target;
+    },
+
+    /** @module OrientationParser */
+    /** Parse features properties, and add position and quaternion attributes.
+     * @function parse
+     * @param {Object} features - a JSON array of Point Features
+     * @param {Object} options - additional properties.
+     * @param {string} options.crsOut - the CRS of View (must be specified (as 'EPSG:4978') for GlobeView)
+     * @return {Object} a promise that resolves with the features given in parameter
+     */
+    parse(features, options = {}) {
```

Comment:
use directly `crsOut` in parameter:
`parse(features, crsOut)`

Reply:
you could move `OrientationParser.parse` in `GeoJsonParser`, and compute quaternion if you have attitude in properties and if type of feature is `point`. So can delete `parse` function from  `OrientationParser` and rename it `OrientationTools`